packages/tools/generator-pie-component/src/app/templates/src/index.ts

```diff
@@ -11,7 +11,7 @@ const componentSelector = 'pie-<%= fileName %>';
 
 export class Pie<%= componentName %> extends LitElement implements <%= componentName %>Props {
```

Comment:
Do we want to consider extending the RTL mixin here because all or most components will need this.

Reply:
we can consider adding it in a separate ticket if we think it's feasible.

Reply:
yes will add it when i review Jamie's suggestion's is the toggle branch